packages/cli/lib/cmds/lint.js

```diff
@@ -19,7 +19,7 @@
 const {cli} = require('@ampproject/toolbox-linter');
 
 function lint(args, logger) {
-  return cli(['dummy'].concat(args), logger, 'amp lint'); // "dummy" to simulate process.argv
+  return cli(['dummy'].concat(args).concat(['-n']), logger, 'amp lint'); // "dummy" to simulate process.argv
```

Comment:
Am I right that this will by default supress the passing tests? How do I print the passing tests? Do I need to set `-n=false`? I'd find it to be more intuitive to name the attribute something like `--show-passing` which is disabled by default.

Reply:
`-n` will unsupress the passing tests since they are hidden by default. I had to add this flag here to pass the toolbox testing suite, as one of its checks is "has at least one passing test". 
I can change the flag to `-s, --show-passing`